utils/loader.py

```diff
@@ -0,0 +1,58 @@
+import os
+import pandas as pd
+from skimage import io
+from torch.utils.data import Dataset
+
+class HuaweiDataset(Dataset):
+    def __init__(self, root_dir=None, transform=None):
+        """
+        Args:
+            root_dir (string, optional): Directory with all the image folders, and 'Training_Info.csv'.
+            transform (callable, optional): Optional transform to be applied on a sample.
+        """
+        self.root_dir = root_dir
+        self.transform = transform
+        if root_dir is not None:
+            self.root_dir = root_dir
+        else:
+            # Use the default path, assumes repo was cloned alongside a `data` folder
+            self.root_dir = os.path.join(
+                os.path.dirname(os.path.dirname(os.path.dirname(os.path.abspath(__file__)))),
```

Comment:
would be shorter with the new object-oriented path library in python 3.5:

```
from pathlib import Path
self.root_dir = Path(__file__).resolve().parent.parent.parent / Path("data")
```

Reply:
I didn't know that was a thing, I'll redo the paths with that for readability